### ISSUE TITLE
Publish for linux as a snap?

### ISSUE THREAD

**Issue by phargogh** [natcap/invest-workbench#38]
Looking through the `electron-builder` documentation, it looks like the `snap` format is supported.  If this is the case, maybe we could address natcap/invest#121 as a feature of the electron-build process?

**Comment by davemfish** [comment]
Yes, for sure! I haven't tried building a `snap` yet, but I have been building an `AppImage` for linux. I've not done the research to learn how those formats differ.

**Comment by phargogh** [comment]
Nor have I!  And I was looking through my previous notes and found something about distributing InVEST as a [FlatPak](https://flatpak.org/) binary, not that that's supported by `electron-builder` at this time.  Seems like (still) a bit of a wild west of linux packaging

**Comment by davemfish** [comment]
We'll need to put this on hold until we start building invest binaries for linux as part of the invest build process (now separate from the workbench build process)
